Nick/improvements

NickTrinh opened the pull request:
Add error handling mechanism to display a toast notification when text limit is exceeded.

* [`client/pages/toolbox.js`](diffhunk://#diff-a7b97ff9f676afb51c8a08a6eca829ae6085be3b929ac43325afd3497f7f656cR121-R123): Added a condition to check if the error message includes "exceeds limit" and display a toast notification with the error message.

NickTrinh committed:
improvements to profile picture quality

NickTrinh committed:
Merge branch 'nick/improvements' of https://github.com/OHOS-Media/fordham-ai-career-toolbox into nick/improvements

bachdumpling committed:
clean up

bachdumpling committed:
Enhance profile picture quality by standardizing image size in passport.js

NickTrinh committed:
Merge branch 'main' of https://github.com/OHOS-Media/fordham-ai-career-toolbox into nick/improvements

NickTrinh committed:
Merge branch 'nick/improvements' of https://github.com/OHOS-Media/fordham-ai-career-toolbox into nick/improvements

NickTrinh committed:
react hot toast error added for text limit error

bachdumpling commented:
It works great! 
However, can you apply the same mechanism to the resume enhancement step?

NickTrinh committed:
added checks for resume enhancer tool

NickTrinh commented:
Done. Added checks for the resume enhancer tool. Also do you want to change up the prompt for the tool as well? I added the prompts from the document some while ago in config/constants.js.

NickTrinh committed:
added new system prompt for resume enhancer + fixed a minor bug in validateText middleware

bachdumpling committed:
refine resume enhancer prompts to prioritize hard and soft skills keywords extraction
